slapo/build.py

```diff
@@ -155,7 +155,7 @@ def _consolidate_and_broadcast(sch: Schedule):
                 param.orig_shape if hasattr(param, "orig_shape") else param.shape
             )
             new_param = nn.Parameter(
-                torch.empty(orig_shape, dtype=param.dtype, device=local_rank)
+                torch.zeros(orig_shape, dtype=param.dtype, device=local_rank)
```

Comment:
For this case, the parameter will be uninitialized if using `torch.empty`.
https://github.com/huggingface/transformers/blob/main/src/transformers/models/bert/modeling_bert.py#L693

Reply:
Notice the parameter is also not handled by `_init_weights`, which may lead to possible incorrect results since we first put it on the meta device.
https://github.com/huggingface/transformers/blob/main/src/transformers/models/bert/modeling_bert.py#L748-L762